wire-compiler/src/test/java/com/squareup/wire/schema/WireRunTest.kt

```diff
@@ -20,6 +20,9 @@ import com.google.common.jimfs.Jimfs
 import com.squareup.wire.StringWireLogger
 import com.squareup.wire.kotlin.RpcCallStyle
 import com.squareup.wire.kotlin.RpcRole
+import com.squareup.wire.testing.add
```

Comment:
I don't think these imports are required

Reply:
I moved files out of this package, so they used to not need imports and now they do!